apps/notifications/sendNotifications.js

```diff
@@ -0,0 +1,43 @@
+const execSync = require('child_process').execSync;
+const { JsonFile, ConsoleTerminalProvider, Terminal } = require('@rushstack/node-core-library');
+
+// the branch will be changed to main once pr is in
+const notificationResponse = execSync(
+  'git cat-file blob refs/remotes/origin/zhas/cli-notification:common/config/notifications/notifications.json'
```

Comment:
The `refs/remotes/origin/zhas/cli-notification` bit needs to come from `process.argv`, since it's a local testing override.